viewsuspensions Error

TSError: ⨯ Unable to compile TypeScript:
src/commands/suspensions/viewsuspensions.ts.ts:31:21 - error TS2322: Type '{ type: "role"; id: string[]; value: true; }' is not assignable to type 'CommandPermission'.
  Object literal may only specify known properties, and 'id' does not exist in type 'CommandPermission'.

31                     id: config.permissions.ranking,

**FIX**

```
import { discordClient, robloxClient, robloxGroup } from '../../main';
import { CommandContext } from '../../structures/addons/CommandAddons';
import { Command } from '../../structures/Command';
import {
    getInvalidRobloxUserEmbed,
    getRobloxUserIsNotMemberEmbed,
    getSuccessfulUnsuspendEmbed,
    getUnexpectedErrorEmbed,
    getVerificationChecksFailedEmbed,
    getRoleNotFoundEmbed,
    getNotSuspendedEmbed,
    getAlreadySuspendedEmbed,
    noSuspendedRankLog,
    mainColor,
} from '../../handlers/locale';
import { config } from '../../config';
import { provider } from '../../database/router';
import { MessageEmbed } from 'discord.js';


class ViewSuspensionsCommand extends Command {
    constructor() {
        super({
            trigger: 'viewsuspensions',
            description: 'Allows the viewing of all concurrent suspensions.',
            type: 'ChatInput',
            module: 'roblox',
            permissions: [
                {
                    type: 'role',
                    id: config.permissions.ranking,
                    value: true,
                }
            ]
        });
    }

    async run(ctx: CommandContext) {
        if(!config.database.enabled) return ctx.reply({ embeds: [ getUnexpectedErrorEmbed() ] });
        let isThere;
        const suspensions = await provider.findSuspendedUsers();
        let mainEmbed = new MessageEmbed();
        mainEmbed.setTimestamp();
        mainEmbed.setColor(mainColor);
        mainEmbed.setTitle('Current Suspensions');
        for (var i in suspensions) {
            isThere = true;
            const user = await robloxClient.getUser(suspensions[i].robloxId);

            mainEmbed.addField(user.name,`Expires on ${suspensions[i].suspendedUntil.toDateString()}`);
        }
        if (isThere == false) mainEmbed.setDescription("**No Current Suspensions!**");
        return ctx.reply({embeds: [mainEmbed]});
    }
}

export default ViewSuspensionsCommand;

```

--- a/ViewSuspensionPlugin/viewsuspensions.ts
+++ b/ViewSuspensionPlugin/viewsuspensions.ts
@@ -28,7 +28,7 @@ class ViewSuspensionsCommand extends Command {
             permissions: [
                 {
                     type: 'role',
-                    id: config.permissions.ranking,
+                    ids: config.permissions.ranking,
                     value: true,
                 }
             ],

